x-pack/metricbeat/metricbeat.reference.yml

```diff
@@ -158,6 +158,21 @@ metricbeat.modules:
   period: 10s
   hosts: ["localhost:3000"]
 
+  # Aerospike Cluster Name
+  #cluster_name: myclustername
+
+  # Optional SSL/TLS. By default is off.
```

Comment:
```suggestion
  # Optional SSL/TLS (disabled by default)
```

Reply:
Make similar changes everywhere in this PR.

Reply:
Resolved with https://github.com/elastic/beats/pull/38126/commits/4a15b5bcb10643bc3e5acafaae0d46a3b9eda521